lms/djangoapps/program_enrollments/api/v1/views.py

```diff
@@ -1389,3 +1397,62 @@ def _calc_response_code(grade_results):
         if any(result.is_error for result in grade_results):
             return status.HTTP_207_MULTI_STATUS
         return status.HTTP_200_OK
+
+
+class EnrollmentDataResetView(APIView):
+    """
+    Resets enrollments and users for a given organization and set of programs.
+    Note, this will remove ALL users from the input organization.
+
+    Path: ``/api/program_enrollments/v1/integration-reset/``
+
+    Accepts: [POST]
+
+    ------------------------------------------------------------------------------------
+    POST
+    ------------------------------------------------------------------------------------
+
+    **Returns**
+        * 200: OK - Enrollments and users sucessfully deleted
+        * 400: Bad Requeset - Program does not match the requested organization
+        * 401: Unauthorized - The requesting user is not authenticated.
+        * 404: Not Found - A requested program does not exist.
+
+    **Response**
+    """
+    authentication_classes = (
+        JwtAuthentication,
+        OAuth2AuthenticationAllowInactiveUser,
+        SessionAuthenticationAllowInactiveUser,
+    )
+    permission_classes = (permissions.JWT_RESTRICTED_APPLICATION_OR_USER_ACCESS,)
+
+    @transaction.atomic
+    def post(self, request):
```

Comment:
Nice. I like how you were able to re-use management commands in order to make this function really straightforward.